snippets/header-drawer.liquid

```diff
@@ -110,7 +110,19 @@
               {%- endfor -%}
             </ul>
           </nav>
-          <div class="menu-drawer__utility-links">
+          {%- liquid
+            assign social_links = false
+            assign localization_forms = false
+
+            if settings.social_twitter_link != blank or settings.social_facebook_link != blank or settings.social_pinterest_link != blank or settings.social_instagram_link != blank or settings.social_youtube_link != blank or settings.social_vimeo_link != blank or settings.social_tiktok_link != blank or settings.social_tumblr_link != blank or settings.social_snapchat_link != blank
+              assign social_links = true
+            endif
+
+            if localization.available_countries.size > 1 and section.settings.enable_country_selector or section.settings.enable_language_selector and localization.available_languages.size > 1
+              assign localization_forms = true
+            endif
```

Comment:
This one I believe should work if I add variables that will store
`localization.available_countries.size > 1 and section.settings.enable_country_selector section.settings.enable_language_selector and localization.available_languages.size > 1`
to theme.liquid then we can reuse it for both the header and footer.

Reply:
Oh actually here it would be just for the header. Cause we're checking on the `section.settings`. 

So you could add it to `header.liquid` and pass in that value to the `header-drawer` snippet if you need to reference it there as well.

Reply:
We're not adding this ? I don't mind the way it currently is. We can always re work it later. It's not hard to read in its current state.